Doppler.Sap/Factory/SapTaskHandler.cs

```diff
@@ -10,27 +10,34 @@
 using Newtonsoft.Json;
 using Microsoft.Extensions.Options;
```

Comment:
This `using ` sentence is not necessary because you remove inline 25 from the IOption interface

Reply:
Done